### ISSUE TITLE
Android builds broken due to android SDK change

### ISSUE THREAD

**Issue by corcoran** [rudderlabs/rudder-sdk-flutter#34]
Incorrect use of semantic versioning and lack of version pinning has broken all Flutter on Android builds that use this library.

Caused by this commit with breaking changes: https://github.com/rudderlabs/rudder-sdk-android/commit/8bde3a27d32ede0387088cebde01aa340edfae6a

Error:
```
/home/.pub-cache/hosted/pub.dartlang.org/rudder_sdk_flutter-1.0.7/android/src/main/java/com/rudderstack/sdk/flutter/RudderSdkFlutterPlugin.java:211: error: cannot find symbol
                RudderClient.updateWithAdvertisingId(
                            ^
  symbol:   method updateWithAdvertisingId(String)
  location: class RudderClient
/home/.pub-cache/hosted/pub.dartlang.org/rudder_sdk_flutter-1.0.7/android/src/main/java/com/rudderstack/sdk/flutter/RudderSdkFlutterPlugin.java:221: error: cannot find symbol
                RudderClient.setAnonymousId((String) argumentsMap.get("anonymousId"));
                            ^
  symbol:   method setAnonymousId(String)
  location: class RudderClient
```

**Comment by corcoran** [comment]
@desusai7

**Comment by desusai7** [comment]
Hey @corcoran , we will fix this in the next release and also we will start pinning a specific version of Android SDK across all our cross platform SDK's

Thank you

**Comment by arnab-p** [comment]
@corcoran These fixes have been released in version `1.2.1`. Please feel free to reopen the issue if it doesn't work.
